modules/round/src/main/PerfsUpdater.scala

```diff
@@ -123,16 +123,22 @@ final class PerfsUpdater(
       correspondence = perfs.correspondence.toRating
     )
 
-  private def updateRatings(white: glicko2.Rating, black: glicko2.Rating, game: Game): Unit =
+  private def updateRatings(
+      advantage: Double,
+      white: glicko2.Rating,
+      black: glicko2.Rating,
+      game: Game
+  ): Unit =
     val results = glicko2.GameRatingPeriodResults(
       List(
         game.winnerColor match
-          case None              => glicko2.GameResult(white, black, true)
-          case Some(chess.White) => glicko2.GameResult(white, black, false)
-          case Some(chess.Black) => glicko2.GameResult(black, white, false)
```

Comment:
Here we go from
```scala
          case Some(chess.White) => glicko2.GameResult(white, black, false)
          case Some(chess.Black) => glicko2.GameResult(black, white, false)
          ```
to
```scala
          case Some(chess.White) => glicko2.GameResult(white, black, Some(true))
          case Some(chess.Black) => glicko2.GameResult(black, white, Some(false))
          ```
which I'm pretty sure has a bug, not fixed by further commits

Reply:
I've made a best effort to correct this (and pushed my commits), and am overwhelmed trying to understand compile errors.